### PR TITLE
002 unit test basic functionality

### PR DESCRIPTION
**Pull request by Francesca-E**

### CODE REVIEW

File: pages/index.test.js [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    ],
+    [
+      { id: 456, content: "ghi" },
+      { id: 789, content: "jkl" },
+    ],
+  ];
+  const updateBoard = jest.fn();
+  const setInputValue = jest.fn();
+  storeItems(updateBoard, "test content", initialBoard, setInputValue);
+  expect(updateBoard).toBeCalled();
```
Comment (jameshopkins) [inline pages/index.test.js:130]: I reckon, like the other tests, we should assert that the argument to `updateBoard` should be of a certain structure

**Review by jameshopkins — CHANGES_REQUESTED** [review]
Not a blocker now, but I think it would be worth removing the mutations on a couple of the functions dealing with data structures, like we did with `handleRemoveItem()` earlier.

File: pages/index.test.js (hunk 1 above)
Comment (jameshopkins) [reply pages/index.test.js:130]: Ignore this. The creation of the ID here will eventually be replaced by the result of the DB record creation, so I wouldn't worry about it for now 👍

**Review by jameshopkins — APPROVED** [review]

**Review by ruiramos — APPROVED** [review]
Looks great, well done! Love what you're doing with passing all the functions the handlers need, it makes it really easy to test and change the internal behaviour in the future.

File: pages/index.test.js (hunk 1 above)
Comment (ruiramos) [reply pages/index.test.js:130]: You could maybe expect the length of the backlog board to have increased by 1?

**Pull request merged**